configure.ac

```diff
@@ -5971,6 +5974,7 @@ AC_PLUGIN([fscache],             [$plugin_fscache],         [fscache statistics]
 AC_PLUGIN([gmond],               [$with_libganglia],        [Ganglia plugin])
 AC_PLUGIN([grpc],                [$with_grpc],              [gRPC plugin])
 AC_PLUGIN([hddtemp],             [yes],                     [Query hddtempd])
+AC_PLUGIN([hugepages],           [yes],                     [Hugepages statistics])
```

Comment:
Since this is a Linux-only plugin, it should only be enabled when being compiled on Linux. Search `configure.ac` for `plugin_cgroups` which is a good example of how to do that.